dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -752,6 +752,10 @@ info.checkpoint.restore.failed=CWWKE0957I: Restoring the checkpoint server proce
 info.checkpoint.restore.failed.explanation=An error occurred restoring a checkpoint server process.  The server launched without using the checkpoint image.
 info.checkpoint.restore.failed.useraction=Check the logs to determine why the checkpoint process was not restored.
 
+error.checkpoint.securitymanager.not.supported=CWWKE0958E: The server checkpoint request failed because the websphere.java.security property was set in the bootstrap.properties file.  This property enables the Java Security Manager and is not valid when doing a server checkpoint.
+error.checkpoint.securitymanager.not.supported.explanation=As part of the deprecation of the Java Security Manager, as specified by JEP 411, Liberty does not support the Java Security Manager when a server checkpoint is requested.
```

Comment:
Reword to "Liberty does not support the Java Security Manager when a server checkpoint is requested. The Java Security Manager is deprecated, as specified by JEP 411."

Reply:
done